.github/workflows/ci-aws.yml

```diff
@@ -135,6 +136,58 @@ jobs:
         id: test
         run: lerna run --scope=${{matrix.package}} --loglevel=silent test
 
+      - name: Run Code coverage 
+        if: github.event_name == 'pull_request'
+        id: cc
+        run: lerna run --scope=${{matrix.package}} --loglevel=silent test:cov
+      
+      - name: Coverage Report
+        if: github.event_name == 'pull_request'
+        uses: danhunsaker/clover-reporter-action@v0.2.17-clover
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          clover-file: packages/${{matrix.service}}/coverage/clover.xml
+      
+      - name: Find Comment Coverage Report
+        if: github.event_name == 'pull_request'
+        uses: peter-evans/find-comment@v2
+        id: fc
+        with:
+          issue-number: ${{ github.event.number }}
+          body-includes: "Total Coverage:"
+          direction: last
```

Comment:
you can get a race condition on it (last comment) and get by mistake the wrong code cov -you can maybe output the cod coverage reports and write them to the pull request synchronically on a separated step.

Reply:
There is no way to output code-coverage action. the workaround I've made is to create a new comment which created before the code coverage report synchronically on a different job.